frontend/src/elm/Login/Major/Models.elm

```diff
@@ -0,0 +1,22 @@
+module Login.Major.Models exposing (..)
+
+
+type alias Major =
+  { majorType : String
+  , majorName : String
+  }
+
+
+type alias MajorForm =
```

Comment:
I reference https://github.com/sporto/elm-tutorial-app
and this example named player not `Model` but `Player`.
I also think that the name of a 'type' like `MajorForm` and `Player` should be intuitive.
The name `Model` is used to describe the entire model.
The function `view` and `update` can be distinguished by type, or we can change name of function.
And the type `Msg` is proper itself.
So I thick this is good.